lib/wallet/wallet.js

```diff
@@ -1975,8 +1940,8 @@ class Wallet extends EventEmitter {
     if (value > lockup)
       throw new Error(`Bid exceeds lockup value: "${name}".`);
 
-    const addr = !consecutiveCall ? await this.receiveAddress(acct)
-      : (await this.createReceive()).getAddress();
+    const addr = consecutiveCall ? await this.receiveAddress(acct)
```

Comment:
Didn't this cause a bug before? I thought that it's supposed to be `!consecutiveCall` otherwise batch bids don't generate distinct addresses?

Reply:
It should be same as production-namebase which is as you say. 

```
const addr = !consecutiveCall ? await this.receiveAddress(acct)
      : (await this.createReceive()).getAddress();
```

One question is we decided to get rid of this completely is that not so ? (generating new address for each bid in the batch ?)

I am reverting to what we have

Reply:
fixed ...

Reply:
Seems to still be missing `this.createReceive(acct)`

Reply:
added now